snippets/snippets.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Student File Header Template": {
+    "prefix": "!fhead",
+    "body": [
+      "$LINE_COMMENT $RELATIVE_FILEPATH",
+      "$LINE_COMMENT Created on: $CURRENT_MONTH_NAME $CURRENT_DATE $CURRENT_YEAR",
+      "$LINE_COMMENT Student name: ${1:fullname}",
+      "$LINE_COMMENT Student email: ${2:email}@myseneca.ca",
+      "$LINE_COMMENT Class: ${3:classname}",
```

Comment:
Just a minor nitpick but I think the terminology of "Course" would be more appropriate "Class".

Reply:
agree

Reply:
updated